ingestion-core/setup.py

```diff
@@ -32,6 +31,9 @@ def get_long_description():
     python_requires=">=3.8",
     package_dir={"": "src"},
     zip_safe=False,
+    use_incremental=True,
```

Comment:
Oh that's really cool, did not know that!

Reply:
Thanks @pmbrull 👍🏼